src/content/docs/logs/get-started/logging-best-practices.mdx

```diff
@@ -0,0 +1,158 @@
+* Use the **Parsing** UI to test your Grok patterns. You can paste sample logs in the **Parsing** UI to validate your Grok or Regex patterns and if they're extracting the attributes as you expect.
+ * Add anchors to parsing logic (i.e., `^`) to indicate the beginning of a line, or `$` at the end of a line.
+ * Use `()?` around a pattern to identify optional fields
+ * Avoid overusing expensive Grok patterns like `'%{GREEDYDATA}`. Try to always use a valid Grok pattern and Grok type when extracting attributes.
+
+## Drop Filter Rules [#drop-rules]
+
+### Drop logs at ingest
+
+* Create [drop filter rules](/docs/logs/ui-data/drop-data-drop-filter-rules#create) to drop logs that are not useful or that are not required to satisfy any use cases for dashboards, alerts or troubleshooting
+
+### Drop attributes from your logs at ingest
+
+* Create drop rules to drop unused attributes from your logs.
+* Drop the `message` attribute after parsing. If you parse the message attribute to create new attributes from the data, drop the message field.
+* Example: If you're forwarding data from AWS infrastructure, you can create drop rules to drop any AWS attributes that may create unwanted data bloat.
+
+## New Relic Logs Sizing [#sizing]
+
+* How we bill for storage may differ from some of our competitors. How we meter for log data is similar to how we meter and bill for other types of data, which is defined in [Usage calculation](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing#usage-calculation).
+* If our cloud integrations (AWS, Azure, GCP) are installed, we will add cloud metadata to every log record, which will add to the overall ingest bill. This data can be dropped though to reduce ingest. A public screen recording showing how you can drop the AWS metadata can be found [here](https://drive.google.com/file/d/1bqEu2b6CxX3jfcp0ggJDj_2ckquGc7aP/view?usp=sharing).
+* The main drivers for log data overhead are below, in order of impact:
+ * Cloud integrations
+ * JSON formatting
+ * Log patterns (You can disable/enable patterns in the **Logs** UI.)
+
+## Searching logs [#searching-logs]
+
+* For common log searches, create and use **Saved views** in the UI. Create a search for your data and then use the `+ Add Column` selector to add additional relevant columns (attributes) to the table output below. You can then move the columns around so that they appear in the order you want, then save it as a saved view with either private or public permissions.  Create public saved views so that you and other users can easily run common searches with all relevant attribute data displayed. This is good practice for 3rd-party applications like apache, nginx, etc. so users can easily see those logs without searching.
```

Comment:
What's the "table output below"? Does that refer to something on this page or a table in the UI?